src/components/FolderPage/FolderButtons.js

```diff
@@ -0,0 +1,21 @@
+import "./FolderButtons.css";
+
+const FolderButtons = (folderList, handleFolderClick) => (
+  <>
+    <button className="folderButton" onClick={() => handleFolderClick()}>
+      전체
+    </button>
+    {folderList.map((folder) => (
+      <div key={folder.id}>
+        <button
+          className="folderButton"
+          onClick={() => handleFolderClick(folder.id)}
+        >
+          {folder.name}
+        </button>
+      </div>

```

Comment:
여기도 마크업을 간결하게, `div`가 없어도 되겠네요.  

```suggestion
        <button
        key={folder.id}
          className="folderButton"
          onClick={() => handleFolderClick(folder.id)}
        >
          {folder.name}
        </button>
```